subprojects/gradle/room-config/src/main/kotlin/com/avito/android/plugin/RoomConfigPlugin.kt

```diff
@@ -26,15 +31,29 @@ class RoomConfigPlugin : Plugin<Project> {
             target.dependencies.add("testImplementation", "androidx.room:room-testing:$archPersistence")
         }
 
-        target.plugins.withId("kotlin-kapt") {
+        target.plugins.withId("kotlin-kapt") { plugin ->
 
             target.dependencies.add("kapt", "androidx.room:room-compiler:$archPersistence")
 
             target.extensions.getByType<KaptExtension>().run {
                 arguments {
-                    arg("room.schemaLocation", File(target.projectDir, "room-schemas").absolutePath)
+                    arg("room.schemaLocation", File(target.projectDir, "room-schemas-$kaptIncremental").absolutePath)
```

Comment:
It breaks the client. "room-schemas" is a directory in avito modules

Reply:
@ntoskrnl , cc

Reply:
I think that this plugin is mostly for avito (archPersistenceVersion, rooms-schemas, ...).

My proposal:
- Move this straightforward and simple logic to buildscripts in avito. It's only a couple of dependencies and arguments.
- Move RoomIncrementalKaptSupportCheckMode to build checks.
- Remove RoomConfigPlugin

Reply:
Oops, I've changed `room-schemas` for debug purposes. I'll turn it back. Thanks for noticing.

Reply:
I agree with your proposal. Do you want me to do it in this task?